### PR TITLE
feat: created func for bilinear 

### LINKED ISSUE #21366: bilinear

### PR DESCRIPTION
**Pull request by Sanjay8602**
# PR Description

## Related Issue

Closes #21366 

## Checklist

- [x] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Review by Sanjay8602 — COMMENTED** [review]
Deleted import line for bilinear function in test_common

**Review by Sarvesh-Kesharwani — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py [hunk 1]
```diff
@@ -217,6 +218,40 @@ def paddle_unfold_handler(draw, dtype):
 # ------------ #
 
 
+# bilinear function
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.bilinear",
+    dtype_x1_x2_weight_bias=bilinear(
+        dtypes=helpers.get_dtypes("valid", full=False),
+    ),
```
[the remark this answers is not in the document]
Comment (Sanjay8602) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py:225]: Sure I have taken note of it within next 2 days I will sove it

**Comment by Sanjay8602** [comment]
Hey @Sarvesh-Kesharwani please review this changes

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py [hunk 2]
```diff
@@ -217,6 +217,40 @@ def paddle_unfold_handler(draw, dtype):
...
+def test_paddle_bilinear(
+    *,
+    dtype_x1_x2_weight_bias,
+    on_device,
+    fn_tree,
+    backend_fw,
+    frontend,
+    test_flags,
+):
+    dtype, x1, x2, weight, bias = dtype_x1_x2_weight_bias
```
Comment (Sarvesh-Kesharwani) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py:236]: dtype_and_values helper function return max 3 values,
please check the definition of it and use the appropriate helper function for generating the test data.

**Review by Sarvesh-Kesharwani — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py (hunk 2 above)
Comment (Sarvesh-Kesharwani) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py:236]: image

**Comment by Sarvesh-Kesharwani** [comment]
image
Hi @Mr-Array22,
the issue is still not resolved, please go through the shown error again.
image

**Comment by Sanjay8602** [comment]
> image Hi @Mr-Array22, the issue is still not resolved, please go through the shown error again. image

My name is Sanjay and whom did you tag I didnt get it

**Comment by Sanjay8602** [comment]
Trying to solve it

**Pull request closed without merging**